external/llvm-project/mlir/lib/ExecutionEngine/SystemDevices.cpp

```diff
@@ -0,0 +1,41 @@
+//===- RocmSystemDetect.cpp - Detect ROCm devices -------------------------===//
```

Comment:
@sjw36  I guess you forgot to change this comment after copying it from `RocmSystemDetect.cpp`.